[2022-07-22 13:00:36,201] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: tutorial.t2 scheduled__2022-07-12T00:00:00+00:00 [queued]>
[2022-07-22 13:00:36,264] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: tutorial.t2 scheduled__2022-07-12T00:00:00+00:00 [queued]>
[2022-07-22 13:00:36,265] {taskinstance.py:1376} INFO - 
--------------------------------------------------------------------------------
[2022-07-22 13:00:36,266] {taskinstance.py:1377} INFO - Starting attempt 1 of 2
[2022-07-22 13:00:36,267] {taskinstance.py:1378} INFO - 
--------------------------------------------------------------------------------
[2022-07-22 13:00:36,366] {taskinstance.py:1397} INFO - Executing <Task(PythonOperator): t2> on 2022-07-12 00:00:00+00:00
[2022-07-22 13:00:36,389] {standard_task_runner.py:52} INFO - Started process 901 to run task
[2022-07-22 13:00:36,427] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'tutorial', 't2', 'scheduled__2022-07-12T00:00:00+00:00', '--job-id', '91', '--raw', '--subdir', 'DAGS_FOLDER/nickdag.py', '--cfg-path', '/tmp/tmp9l1k8z0q', '--error-file', '/tmp/tmpvato3zhc']
[2022-07-22 13:00:36,432] {standard_task_runner.py:80} INFO - Job 91: Subtask t2
[2022-07-22 13:00:36,494] {logging_mixin.py:115} WARNING - /home/***/.local/lib/python3.7/site-packages/***/configuration.py:528 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2022-07-22 13:00:36,739] {task_command.py:371} INFO - Running <TaskInstance: tutorial.t2 scheduled__2022-07-12T00:00:00+00:00 [running]> on host 2f63307804d5
[2022-07-22 13:00:37,211] {taskinstance.py:1591} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_EMAIL=caroline.kosalka@gmail.com
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=tutorial
AIRFLOW_CTX_TASK_ID=t2
AIRFLOW_CTX_EXECUTION_DATE=2022-07-12T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-07-12T00:00:00+00:00
[2022-07-22 13:00:37,216] {python.py:173} INFO - Done. Returned value was: goodbye
[2022-07-22 13:00:37,360] {taskinstance.py:1420} INFO - Marking task as SUCCESS. dag_id=tutorial, task_id=t2, execution_date=20220712T000000, start_date=20220722T130036, end_date=20220722T130037
[2022-07-22 13:00:37,436] {local_task_job.py:156} INFO - Task exited with return code 0
[2022-07-22 13:00:37,786] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2022-07-25 09:18:15,395] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: tutorial.t2 scheduled__2022-07-12T00:00:00+00:00 [queued]>
[2022-07-25 09:18:15,409] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: tutorial.t2 scheduled__2022-07-12T00:00:00+00:00 [queued]>
[2022-07-25 09:18:15,410] {taskinstance.py:1376} INFO - 
--------------------------------------------------------------------------------
[2022-07-25 09:18:15,410] {taskinstance.py:1377} INFO - Starting attempt 1 of 2
[2022-07-25 09:18:15,411] {taskinstance.py:1378} INFO - 
--------------------------------------------------------------------------------
[2022-07-25 09:18:15,430] {taskinstance.py:1397} INFO - Executing <Task(PythonOperator): t2> on 2022-07-12 00:00:00+00:00
[2022-07-25 09:18:15,436] {standard_task_runner.py:52} INFO - Started process 834 to run task
[2022-07-25 09:18:15,443] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'tutorial', 't2', 'scheduled__2022-07-12T00:00:00+00:00', '--job-id', '84', '--raw', '--subdir', 'DAGS_FOLDER/nickdag.py', '--cfg-path', '/tmp/tmpid2xbyhm', '--error-file', '/tmp/tmp424lw1dt']
[2022-07-25 09:18:15,446] {standard_task_runner.py:80} INFO - Job 84: Subtask t2
[2022-07-25 09:18:15,474] {logging_mixin.py:115} WARNING - /home/***/.local/lib/python3.7/site-packages/***/configuration.py:528 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2022-07-25 09:18:15,564] {task_command.py:371} INFO - Running <TaskInstance: tutorial.t2 scheduled__2022-07-12T00:00:00+00:00 [running]> on host 01e6b1df1873
[2022-07-25 09:18:15,673] {taskinstance.py:1591} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_EMAIL=caroline.kosalka@gmail.com
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=tutorial
AIRFLOW_CTX_TASK_ID=t2
AIRFLOW_CTX_EXECUTION_DATE=2022-07-12T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-07-12T00:00:00+00:00
[2022-07-25 09:18:15,676] {python.py:173} INFO - Done. Returned value was: goodbye
[2022-07-25 09:18:15,714] {taskinstance.py:1420} INFO - Marking task as SUCCESS. dag_id=tutorial, task_id=t2, execution_date=20220712T000000, start_date=20220725T091815, end_date=20220725T091815
[2022-07-25 09:18:15,775] {local_task_job.py:156} INFO - Task exited with return code 0
[2022-07-25 09:18:15,836] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2022-07-25 12:28:28,328] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: tutorial.t2 scheduled__2022-07-12T00:00:00+00:00 [queued]>
[2022-07-25 12:28:28,397] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: tutorial.t2 scheduled__2022-07-12T00:00:00+00:00 [queued]>
[2022-07-25 12:28:28,399] {taskinstance.py:1376} INFO - 
--------------------------------------------------------------------------------
[2022-07-25 12:28:28,401] {taskinstance.py:1377} INFO - Starting attempt 1 of 2
[2022-07-25 12:28:28,402] {taskinstance.py:1378} INFO - 
--------------------------------------------------------------------------------
[2022-07-25 12:28:28,515] {taskinstance.py:1397} INFO - Executing <Task(PythonOperator): t2> on 2022-07-12 00:00:00+00:00
[2022-07-25 12:28:28,541] {standard_task_runner.py:52} INFO - Started process 1032 to run task
[2022-07-25 12:28:28,612] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'tutorial', 't2', 'scheduled__2022-07-12T00:00:00+00:00', '--job-id', '166', '--raw', '--subdir', 'DAGS_FOLDER/nickdag.py', '--cfg-path', '/tmp/tmp9n_y5u74', '--error-file', '/tmp/tmp8hqahlmq']
[2022-07-25 12:28:28,625] {standard_task_runner.py:80} INFO - Job 166: Subtask t2
[2022-07-25 12:28:28,747] {logging_mixin.py:115} WARNING - /home/***/.local/lib/python3.7/site-packages/***/configuration.py:528 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2022-07-25 12:28:29,128] {task_command.py:371} INFO - Running <TaskInstance: tutorial.t2 scheduled__2022-07-12T00:00:00+00:00 [running]> on host 929e6a3728d4
[2022-07-25 12:28:29,718] {taskinstance.py:1591} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_EMAIL=caroline.kosalka@gmail.com
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=tutorial
AIRFLOW_CTX_TASK_ID=t2
AIRFLOW_CTX_EXECUTION_DATE=2022-07-12T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-07-12T00:00:00+00:00
[2022-07-25 12:28:29,721] {python.py:173} INFO - Done. Returned value was: goodbye
[2022-07-25 12:28:29,954] {taskinstance.py:1420} INFO - Marking task as SUCCESS. dag_id=tutorial, task_id=t2, execution_date=20220712T000000, start_date=20220725T122828, end_date=20220725T122829
[2022-07-25 12:28:30,060] {local_task_job.py:156} INFO - Task exited with return code 0
[2022-07-25 12:28:30,290] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
